<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>161</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="length">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Length</string>
     </property>
    </widget>
    <widget class="QLabel" name="width">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
    <widget class="QLabel" name="price_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rec_length">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="rec_width">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="rec_price">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Compare</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="diameter">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Diameter</string>
     </property>
    </widget>
    <widget class="QLabel" name="price">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rnd_price">
     <property name="geometry">
      <rect>
       <x>62</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="rnd_diameter">
     <property name="geometry">
      <rect>
       <x>62</x>
       <y>0</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="result_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>comparePizzas()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>comparePizzas()</slot>
 </slots>
</ui>
